src/app/components/file-permissions-modal/file-permissions-modal.component.scss

```diff
@@ -68,6 +68,28 @@
   grid-row-start: 1;
   grid-row-end: 2;
 }
+
+.modal-column-full-width {
+  width: 100%;
```

Comment:
This is generic modal CSS that can be re-used so it can sit in 
`src/app/shared/modal.component.scss`

Reply:
Fixed.